src/codegen/codegen_vhls.cc

```diff
@@ -67,6 +68,42 @@ void CodeGenVivadoHLS::PreFunctionBody(LoweredFunc f) {
   this->stream << "#pragma HLS INTERFACE s_axilite port=return bundle=control\n\n";
 }
 
+void CodeGenVivadoHLS::VisitExpr_(const Min *op, std::ostream& os) {  // NOLINT(*)
+  const char *opstr = "std::min";
+  if (op->type.is_float()) {
+    switch (op->type.bits()) {
+      case 32:
+        opstr = "fminf"; break;
+      case 64:
+        opstr = "fmin"; break;
+    }
+  }
+
+  os << opstr << '(';
+  PrintExpr(op->a, os);
+  os << ", ";
+  PrintExpr(op->b, os);
+  os << ')';
```

Comment:
Could you introduce `PrintBinaryExpr` to avoid boilerplate code for binary expression? 
c.f. https://github.com/dmlc/tvm/blob/master/src/codegen/codegen_c.cc#L372